### DISCUSSION TITLE
Cannot creat more than 6 participants on Windows.

### DISCUSSION THREAD

**Discussion by alexmain2901** [eProsima/Fast-DDS#2687] [Support]
I am having an issue where creating more than 6 participants on Windows causes the creat participant function to hang. The same code works fine on Linux (Rhel 7 & 8). Has anyone ever come across an issue like this?

Thank you for your time.

**Comment by JLBuenoLopez** [comment]
Hi @alexmain2901,

Fast DDS developers team is not aware of any issue of that kind. Fast DDS CI tests Participant creation in every Tier 1 platform among which Windows 10 is included. In one specific test more than 20 Participants are created, so I do not think the issue is related with Participant creation but maybe with some configuration setting.
